dataplicity/m2m/packetbase.py

```diff
@@ -51,13 +51,24 @@ def __init__(self, *args, **kwargs):
 
     def __repr__(self):
         data = {}
-        for attrib_name, attrib_type in self.attributes:
+        for attrib_name, _attrib_type in self.attributes:
             try:
                 data[attrib_name] = getattr(self, attrib_name)
             except AttributeError:
                 continue
 
-        params = ', '.join("{}={!r}".format(k, v if k != 'password' else '********') for k, v in data.items())
+        def summarize(key, value):
```

Comment:
Trims long strings from packet repr, otherwise you get 16K of binary gibberish with wormhole traffic.